packages/api/src/auth/decoders/clerk.ts

```diff
@@ -1,38 +1,28 @@
-import type { APIGatewayProxyEvent, Context as LambdaContext } from 'aws-lambda'
+import type IClerk from '@clerk/clerk-sdk-node/instance'
 
-interface Req {
-  event: APIGatewayProxyEvent
-  context: LambdaContext
-}
-
-export const clerk = async (token: string, req: Req) => {
+export const clerk = async (token: string) => {
   // Use require here, to prevent needing clerk sdk in api deps
-  const { sessions, users } = require('@clerk/clerk-sdk-node')
+  const Clerk = require('@clerk/clerk-sdk-node/instance').default
+
+  const { users, base }: IClerk = new Clerk()
 
   if (!process.env.CLERK_API_KEY) {
     console.error('CLERK_API_KEY env var is not set.')
     throw new Error('CLERK_API_KEY env var is not set.')
   }
 
-  // Clerk sessions are a combination of a clerk "current session id", which we store
-  // in the Redwood auth token, and the __session cookie, which contains a second session
-  // bearer token. The two tokens together define which device is browsing and as who.
-  const clerkCookieName = '__session'
-  const cookies = req.event.headers['cookie']?.split(';').map((c) => c.trim())
-  const sessionCookie = cookies
-    ?.find((c) => c.startsWith(clerkCookieName + '='))
-    ?.substring(clerkCookieName.length + 1)
-
-  if (!sessionCookie || sessionCookie.length < 1) {
-    return Promise.reject(new Error('Clerk __session token is not set'))
+  if (!process.env.CLERK_JWT_KEY) {
+    console.error('CLERK_JWT_KEY env var is not set.')
+    throw new Error('CLERK_JWT_KEY env var is not set.')
   }
 
-  const session = await sessions.verifySession(token, sessionCookie)
-  if (!session.userId) {
+  const jwtPayload = await base.verifySessionToken(token)
+
+  if (!jwtPayload.sub) {
```

Comment:
@Tobbe 
The `verifySessionToken` method will throw errors on its own if verification fails for some reason. The messages of possible errors from an expected execution are:
- 'Token is expired'
- 'Token is not active yet'
- 'Issuer is invalid: {issuer name}'
- 'Authorized party is invalid: {azp name}'
- 'Failed to verify token'

So I would suggest that we wrap this in a `try-catch` block :)